packages/components/src/primitives/Box/Box.tsx

```diff
@@ -6,6 +6,8 @@ import type {
 } from "@xstyled/styled-components";
 
 export interface BoxProps extends SystemProps {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  as?: any;
```

Comment:
It still works as expected by the patience to handle TS abandoned me

![Screen Shot 2025-04-23 at 20 19 24](https://github.com/user-attachments/assets/0194ec08-f54a-4058-8d06-e92dd127e41a)

![Screen Shot 2025-04-23 at 20 19 19](https://github.com/user-attachments/assets/4fcd990a-c886-4c4f-9d10-5b2de7c8bd9f)